README.md

```diff
@@ -16,7 +16,7 @@ A list of companies that have paid Developer Community Writer Programs.
 - [CircleCI](https://circleci.com/blog/guest-writer-program/)  - Up to $300 per piece
   > Technical tutorials with code. Pick from a list of possible articles.
 
-- [Clubhouse.io](https://clubhouse.io/clubhouse-write-earn-give-program/) - Up to $600 per piece
+- Clubhouse.io - Currently returns Error 404 (Not found)
```

Comment:
hey @ziizium thanks for thus, I got a report saying they have closed their program. Could you change the text to reflect that?

Reply:
Done

Reply:
@malgamves It's been a while I made the changes you asked for, anything else?